test/unit/algo/algo_test.hpp

```diff
@@ -0,0 +1,26 @@
+//==================================================================================================
```

Comment:
test.hpp is maybe enough for the name, we know we're in algo/

Reply:
Cannot do that.

Just "test.hpp" will collide with our normal "test.hpp"

"algo/test.hpp" does not work either.

Feel free to try later.

Reply:
ok